build.sh

```diff
@@ -100,30 +117,37 @@ fi
 
 echo 'Building command binary.'
 
-cargo build --release
+if [ "${opts[debug]}" = true ]; then
+    cargo build
+    executable="./target/debug/rs"
+else
+    cargo build --release
+    executable="./target/release/rs"
+fi
 
 echo
 
 if [ ! -f ./target/release/rs ]; then
```

Comment:
This will fail when the `--debug` flag is enabled if the `target/release` directory is not present.
Should probably use something like `if [ ! -f "$executable" ]; then ...` instead.

Reply:
Good catch! I changed references to that path with `s#'./target/resease/rs'#"$executable"`, which missed that case because it wasn't wrapped in single quotes.